plugins/janus_videoroom.c

```diff
@@ -1542,20 +1542,22 @@ struct janus_plugin_result *janus_videoroom_handle_message(janus_plugin_session
 		guint32 video_handle = 0;
 		if(audio_port > 0) {
 			audio_handle = janus_rtp_forwarder_add_helper(publisher, host, audio_port, 0);
+			if(audio_handle)
+				janus_refcount_increase(&publisher->ref);
 		}
 		if(video_port > 0) {
 			video_handle = janus_rtp_forwarder_add_helper(publisher, host, video_port, 1);
+			if(video_handle) {
+				janus_refcount_increase(&publisher->ref);
```

Comment:
```if(video_handle) {``` opens a conditional here but I don't see it being closed, should this be ```if(video_handle)``` instead?